CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+ * [Fixing bugs](#fixing-bugs)
+ * [Filing issues](#filing-issues)
+ * [Running Tests](#running-tests)
+ * [Tips and guidelines](#guidelines-and-tips)
+
+Once your PR is accepted it will incorporated into the next main release of Autoprefixer.
+
+**Important**: Autoprefixer works by pulling data from [caniuse](https://caniuse.com/). Therefore it's important that support for a particular CSS feature is available there before we can enable support for it here. You can check the listing of features that need to be added [here](https://caniuse.com/issue-list).
+
+## Getting started
+Before you begin contributing make sure you have a [GitHub account](https://github.com/signup/free).
+* [Fork the repository](https://github.com/postcssAautoprefixer)
+* Clone a copy of it to your computer: `git clone https://github.com/USERNAME/autoprefixer`
+* run `yarn install` this will install all dependencies needed to run tests
+* File an issue, see [filing issues](#filing-issues)
+* Once issue is discussed and accepted, you can begin making the necessary changes
+
+## Adding a prefix
+We'll explain how would you go about adding a CSS feature to Autoprefixer. For example, we'll add support for a CSS feature called `background-clip: text`
+
+Note: Remember that the feature that you want to add must also be supported on [caniuse](https://caniuse).
+
+* Create a topic branch from the `master` branch. Like this: `git checkout -b add/background-clip-text`
+tip: use 'add', 'fix' or 'enhancement' to indicate on your type of contribution
+
+* To add support for a CSS feature you must call the convert function inside of `data/prefixes.js` and specify three parameters: data, options and a callback. Like this:
+  ```
+  f(require('caniuse-lite/data/features/background-clip-text'), browsers =>
+    prefix(['background-clip'], {
+      feature: 'background-clip-text',
+      browsers
+    })
+  )
+  ```
+* Add a hack to the `libs/hacks` folder using the name of the CSS feature as the filename.
+
+* Create a a class which will extends `Value`. In that hack class change the prefix for IE to -webkit-. See appearance hack. [See complete example](https://github.com/postcss/Autoprefixer/blob/73c7b6ab090a9a9a03869b3099096af00be7eb7d/lib/hacks/background-clip.js)
+
+* Load this hack in `lib/prefixes.js`. [See example](https://github.com/postcss/autoprefixer/blob/73c7b6ab090a9a9a03869b3099096af00be7eb7d/lib/prefixes.js)
+
+* Add tests to `test/autoprefixer.test.js` to test that Autoprefixer works with Chrome and Edge browsers adds only -webkit- prefix. [See details here](https://github.com/postcss/autoprefixer/commit/73c7b6ab090a9a9a03869b3099096af00be7eb7d)
+
+## Changing an existing prefix
+TODO
+
+## Fixing Bugs
+TODO
+
+## Filing issues
+Issues, questions, or comments about existing features should be filed as an issue. Support for new features also requires filing a new issue. You can file an issue [here](https://github.com/postcssAautoprefixer/issues).
+
+## Running Tests
+TODO
```

Comment:
This section should be given priority. People can't contribute until they know how to test their changes.

Reply:
You're right Daniel! You're more than welcome to contribute so we can improve this.